leftwm-core/src/handlers/mouse_combo_handler.rs

```diff
@@ -19,9 +19,17 @@ impl State {
         if let Some(window) = self.windows.iter().find(|w| w.handle == handle) {
             if !self.disable_tile_drag || window.floating() {
                 let modifier = utils::modmask_lookup::into_modmask(&self.mousekey);
+                let win = window.clone();
```

Comment:
```suggestion
                let bottom_right = (window.x() + window.width(), window.y() + window.height());
```

Reply:
Yeah, it is sensible. I prefer this solution too.